### ISSUE TITLE
Multiprocessing

### ISSUE THREAD

**Issue by zimonitrome** [JargeZ/ntscqt#64]
My CPU idles at 10% while taking minutes to render short videos. Does the application use multiprocessing for rendering? Otherwise it could be something to look into.

The standard Python library `multiprocessing` could maybe be used?

**Comment by JargeZ** [comment]
Yes, work in this direction began, but was not brought to an end. There is a issue #26 with a description of the solution
I will implement into this someday or you can help and make a pull request if you have the desire for it ;)
